source/_components/notify.hangouts.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: "Google Hangouts"
+description: "Instructions on how to add Google Hangouts notifications to Home Assistant."
+date: 2018-08-18 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hangouts.png
+ha_category: Notifications
+ha_release: 0.77
+---
+
+
+The `hangouts` platform allows you to deliver notifications from Home Assistant to [Google Hangouts](http://hangouts.google.com) conversations. Conversations can be both direct as well as group chats.
+
+## {% linkable_title Configuration %}
+
+To enable Hangouts notifications in your installation, you first need to configure
+the [Hangouts component](/components/hangouts/). Then, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry  
+notify:
+  - name: NOTIFIER_NAME
+    platform: hangouts
+    default_conversations:
+      - id: CONVERSATION_ID
+      - name: CONVERSAION_NAME
+```
+
+{% configuration %}
+name: 
+  description: "Setting the optional parameter `name` allows multiple notifiers to be created. The default value is `notify`. The notifier will bind to the service `notify.NOTIFIER_NAME`."
+  required: false
+  type: string
+default_conversations:
+  description: "The conversations all messages will be sent to, when no other target is given."
+  required: true
+  type: [map]
+  keys:
+    id:
+      description: "Specifies the id of the conversation."
+      required: false
```

Comment:
How is this optional when `default_conversations` is required?

Reply:
the problem is, one is required, the name or the id, who can i display it?